### PR TITLE
Adding Negative & Positive Infinities to math concepts

### PR DESCRIPTION
**Pull request by Awurama-N**
closes Issue #2396

### CODE REVIEW

File: code/drasil-data/Data/Drasil/Concepts/Math.hs [hunk 1]
```diff
@@ -12,7 +12,7 @@ mathcon :: [ConceptChunk]
 
 mathcon = [angle, area, axis, calculation, cartesian, centre, change, constraint, diameter,
   equation, euclidN, euclidSpace, gradient, graph, law, line, matrix, norm, normal,
-  normalV, number, orient, parameter, perp, perpV, pi_, point, probability, rOfChng,
+  normalV, number, orient, parameter, perp, perpV, pi_, qdNegInf, qdPosInf, point, probability, rOfChng,
```
Comment (JacquesCarette) [inline code/drasil-data/Data/Drasil/Concepts/Math.hs:15]: please omit the `qd` prefix.  Just `negInf` and `posInf`.

**Review by JacquesCarette — CHANGES_REQUESTED** [review]

**Comment by Awurama-N** [comment]
is this more like what you needed done?
if so, then i'll create pull requests for the other concepts as well. 
I wasn't sure whether you needed it to be "NegInf" or "qdNegInf" (since this is what is already in the system).

File: code/drasil-data/Data/Drasil/Concepts/Math.hs (hunk 1 above)
Comment (Awurama-N) [reply code/drasil-data/Data/Drasil/Concepts/Math.hs:15]: Screenshot 2021-05-13 at 7 38 51 AM
there's a warning that there's a parse error with NegInf, after i take out the "qd"?

Comment (JacquesCarette) [reply code/drasil-data/Data/Drasil/Concepts/Math.hs:15]: `negInf`, not `NegInf`. Look at the capitalization. Identifiers in Haskell cannot start with a capital letter, those are reserved for constructors.

**Comment by JacquesCarette** [comment]
You should add to `Quantity` too, at the same time.

**Review by JacquesCarette — APPROVED** [review]

**Pull request merged**